templates/image-builder-clowder.yml

```diff
@@ -14,8 +14,16 @@ objects:
       iqePlugin: image-builder
     deployments:
     - name: service
+      minReplicas: ${{MIN_REPLICAS}}
```

Comment:
This doesn't seem to do anything, it does work if I change it to `replicas` though, can you double check?

Reply:
According to the api spec https://consoledot.pages.redhat.com/clowder/dev/api_reference.html#k8s-api-github-com-redhatinsights-clowder-apis-cloud-redhat-com-v1alpha1-deployment, minReplicas is indeed deprecated.